templates/views/views-view--txex-events--block-3.html.twig

```diff
@@ -50,7 +50,6 @@
 
   {{ exposed }}
   {{ attachment_before }}
-<h2 class="wysiwyg__h2">My Events</h2>
   {{ rows }}
```

Comment:
Hey @mikeethedude . This looks like it's unused right now, but it is part of the theming for a txex_events view that hasn't been merged yet. The site config has not been updated yet, but this theme has been updated. If you need a new blank block for that view, please create block_4. Does that make sense?

Reply:
I'll make a copy of this template and adjust the view display to be block-4 then. Thanks for the heads up @jeff-cardwell